src/scenes/InternalScene.ts

```diff
@@ -17,72 +19,149 @@ export default class InternalScene extends BaseScene {
     }
 
     setup(){
+        this.macrophageArray.forEach(macrophage => {
+            macrophage.gotoAndStop(0);
+            macrophage.velocity = new PIXI.Point(Math.random()*.5, Math.random()*.5);
+        })
 
+        this.bacteriaArray.forEach(bacteria => {
+            bacteria.gotoAndStop(0);
+            bacteria.velocity = new PIXI.Point(Math.random(), Math.random());
```

Comment:
if you add a -1 to the x velocity of the bacteria then you will guarantee that it is moving to the left at the start, making the bacteria head directly towards the macrophage.
`new PIXI.Point(Math.random()-1, Math.random())`